src/openfl/_internal/renderer/opengl/GLShape.hx

```diff
@@ -59,7 +59,7 @@ class GLShape {
 				renderSession.shaderManager.setShader (shader);
 				
 				shader.data.uImage0.input = graphics.__bitmap;
-				shader.data.uImage0.smoothing = renderSession.allowSmoothing;
+				shader.data.uImage0.smoothing = false;
```

Comment:
This doesn't seem to be related to the PR topic? Also does this matter? Because shapes are always rendered scaled, right?

Reply:
This is to avoid blurry text rendering, especially when scaling down. Text is rendered on canvas including smoothing if needed, so the canvas don't need to be rendered smoothed again on GL level. This is actually true for all canvases. It is similar to the cache as bitmap problem we had in the past.

But I agree, it is better, also for upstream pull-requests, if we don't mix this.

Reply:
I was also thinking should I put it separate on in this PR. Decided to put it at this PR because it is required to have sharp rendering when zoomed out. Of course I can split it.